.github/CODEOWNERS

```
@@ -63,6 +63,7 @@ content/en/logs/log_collection/ios.md                            @Datadog/rum-mo
 # Traces
 content/en/tracing/trace_collection/dd_libraries/android.md     @Datadog/rum-mobile @DataDog/documentation
 content/en/tracing/trace_collection/dd_libraries/ios.md         @Datadog/rum-mobile @DataDog/documentation
+content/en/tracing/**/**/*dotnet*                               @DataDog/tracing-dotnet @DataDog/documentation
```

Wouldn't this be better? `**` is supposed to match any path so it would match 1, 2, 3... levels of folder.
```suggestion
content/en/tracing/**/*dotnet*                              @DataDog/tracing-dotnet @DataDog/documentation
```

---

On the the docs I saw I didn't find that, but if this is what gitignore allows then yeah it should work the same way. Thank you!